test/integration/apiserver/apply/reset_fields_test.go

```diff
@@ -298,3 +297,31 @@ func TestApplyResetFields(t *testing.T) {
 		}
 	}
 }
+
+func expectConflict(t *testing.T, objRet *unstructured.Unstructured, err error, dynamicClient dynamic.Interface, resource schema.GroupVersionResource, namespace, name, where string) {
+	if err != nil && strings.Contains(err.Error(), "conflict") {
+		return
+	}
+	which := "returned"
+	var gotten string
+	var err2 error
+	if objRet == nil {
+		which = "subsequently fetched"
+		objRet, err2 = dynamicClient.
```

Comment:
I don't see yet what is the purpose of getting the object again

Reply:
The purpose of returning the object is to give more information about what went wrong.